conformance/tests/httproute-request-header-modifier.go

```diff
@@ -192,3 +194,28 @@ var HTTPRouteRequestHeaderModifier = suite.ConformanceTest{
 		}
 	},
 }
+
+var HTTPRouteInvalidRequestHeaderModifier = suite.ConformanceTest{
+	ShortName:   "HTTPRouteInvalidRequestHeaderModifier",
+	Description: "An HTTPRoute with invalid request header modifiers is not accepted",
+	Manifests:   []string{"tests/httproute-invalid-request-header-modifier.yaml"},
+	Test: func(t *testing.T, suite *suite.ConformanceTestSuite) {
+		ns := "gateway-conformance-infra"
+		gwNN := types.NamespacedName{Name: "same-namespace", Namespace: ns}
+
+		t.Run("HTTPRoutes that do intersect with listener hostnames", func(t *testing.T) {
+			routes := []types.NamespacedName{
+				{Namespace: ns, Name: "invalid-request-header-modifier-multiple-actions"},
+			}
+			for _, route := range routes {
+				headerConflictCond := metav1.Condition{
+					Type:   string(v1beta1.RouteConditionAccepted),
+					Status: metav1.ConditionFalse,
+					Reason: string(v1beta1.RouteReasonUnsupportedValue),
+				}
+
+				kubernetes.HTTPRouteMustHaveCondition(t, suite.Client, suite.TimeoutConfig, route, gwNN, headerConflictCond)
```

Comment:
If we do solve this with webhook validation, we either may not need this conformance test, or we should explicitly require that conformance tests are run without the validating webhook present so we can ensure that controllers still behave well in its absence. I tend to think we should just start with webhook validation, but open to other opinions here.

/cc @youngnick

Reply:
I agree that we probably only want the webhook--we may as well avoid the additional implementation burden. I've updated the docstring to remove references to the condition as such, since these should never reach implementations.